### ISSUE TITLE
Undo and redo

### ISSUE THREAD

**Issue by dougcooper** [saadq/resumake.io#86]
There have been a few instances where undo and redo would have come in handy.

**Comment by saadq** [comment]
Hmm, what specifically should be undoable/redoable? Just the adding/removing of elements inside a section?

**Comment by dougcooper** [comment]
Yes sir. That’s exactly what happened. Actually, I deleted an entry thinking that I was removing the currently selected entry. Except the last item in the list is always removed. Items in the middle of a list cannot be removed. Probably another issue to be added.
